nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -2609,8 +2279,6 @@ define([
                 })
                 .then(() => {
                     // this will not change, so we can just render it here.
-                    showAboutApp();
-                    showAppSpec();
```

Comment:
The HTML elements that these were supposed to populate were removed long ago, so these functions just waste time.